geojson_pydantic/geometries.py

```diff
@@ -6,6 +6,7 @@
 
 from pydantic import BaseModel, Field, ValidationError, validator
 from pydantic.error_wrappers import ErrorWrapper
+from singledispatchmethod import singledispatchmethod
```

Comment:
this needs to be optional 

https://github.com/developmentseed/titiler/blob/master/src/titiler/core/titiler/core/factory.py#L64-L68